packages/nextjs/src/utils/config.ts

```diff
@@ -175,3 +196,23 @@ export function withSentryConfig(
     webpack: newWebpackExport,
   };
 }
+
+/**
+ * Store the future location of the webpack-processed version of `sentry.server.config.js` in a runtime env variable, so
+ * we know where to find it when we want to initialize the SDK.
+ *
+ * @param config The webpack config object for this build
+ */
+function memoizeOutputPath(config: WebpackConfig): void {
+  const builtServerSDKInitPath = path.join(config.output.path, SERVER_SDK_INIT_PATH);
+  const projectDir = config.context;
+  // next will automatically set variables from `.env.local` in `process.env` at runtime
+  const envFilePath = path.join(projectDir, '.env.local');
+  const envVarString = `SENTRY_SERVER_INIT_PATH=${builtServerSDKInitPath}\n`;
```

Comment:
Can we extract `SENTRY_SERVER_INIT_PATH` to a variable and not be hardcoded?

Reply:
How do you mean? It _is_ a variable, based on the absolute path of the output directory, which is retrieved at (build) runtime.

Reply:
The value of the variable, yes; the name of the variable, no. I mean specifically `SENTRY_SERVER_INIT_PATH`, the left side of the assignment.

Reply:
Oh, you mean so that you'd have to pass it into the function? To Abhi's point above, that would make sense if this is a "set some env vars" function, which it could be - I was just thinking about it more as a "store this particular piece of data" function.